apps/autostart-stop.html.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: Automatically Stop and Start App Machines
+objective: 
+layout: docs
+nav: firecracker
+order: 65
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+You can decide whether you want to automatically stop, or downscale, instances of your app when they're idle and automatically restart them when they're needed. Each instance of your app is running on a Fly machine. Fly Proxy controls automatically starting and automaticaly stopping machines according to settings in your `fly.toml` file, in either the `services` or `http_service` sections:
+
+```toml
+...
+[[services]]
+  autostart = true
+  autostop = true
+...
+```
+
+```toml
+...
+[http_service]
+  internal_port = 8080
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+...
+```
+
+## Default and recomended values
+
+New V2 apps are automatically started and automatically stopped by default:
```

Comment:
Actually, new V2 apps created by `fly launch` since flyctl v0.0.527 will get these fields for its generated fly.toml

Reply:
Should we just say "created by `fly launch`" then, or do we usually include versions like that?

Reply:
I will leave @catflydotio to answer that 🐱

Reply:
I think there's more to say here.

If this part of `fly launch` hasn't changed recently: 

Launching creates a config, and writes it into `fly.toml`; the Machines in the app get configured according to that config if you let `fly launch` carry on through deployment.

But if you say `N` to "deploy now?" the Machines are configured according to what's in `fly.toml` at the time.
You can edit `fly.toml` before the first deployment, and the app will have whatever settings you specified.

Incidentally, this has been kind of a gotcha: `fly launch` writes you a `fly.toml` and is waiting for you to tell it whether to deploy or not, and you can edit and save `fly.toml`, but it's got a config in memory already and doesn't read `fly.toml` again before it deploys.

All that to say: what `fly launch` puts into a new app's config is one kind of default.

You can create an app without `fly launch` (`fly apps create` and write your own `fly.toml`) and it won't be configured that way automatically.

Once people understand that, it's not too serious which version they're using (these settings have only been generated at all for like 5 minutes, anyway, right?). I don't object to putting version numbers in, if you think it's important.

All that to say: what about 

```
`fly launch` configures new V2 apps to start and stop automatically:
```

(Or put it passively to avoid putting `fly launch` at the start of the sentence if you like; I find it useful to just let commands go at the starts of sentences)